setup.cfg

```diff
@@ -43,6 +43,7 @@ prefect_requires =
 ui_requires = 
 	dash<=2.3.1,>=2.0.0
 	dash-bootstrap-components<=1.1.0,>=1.0.0
+	kaleido
```

Comment:
don't need this - its just a dev requirement, not a requirement of the actual package

Reply:
done!